ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/health.yml

```diff
@@ -16,7 +16,7 @@ mas_appws_spec:
       dbSchema: "{{ mas_app_settings_db2_schema }}"
       maxinst:
         demodata: "{{ mas_app_settings_demodata | bool }}"
-        db2Vargraphic: true
+        db2Vargraphic: "{{ mas_app_settings_db2vargraphic | default('true', true) | bool }}"
```

Comment:
`mas_app_settings_db2vargraphic` needs to be added to https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/suite_app_config/vars/manage.yml (and health.yml) otherwise it won't be defined.

In this file it should simply be using "{{ mas_app_settings_db2vargraphic }}" with the default handled in the other file per other params.

Reply:
(Same comment applies to the changes in health as well)

Reply:
applied